.github/workflows/build.yml

```diff
@@ -0,0 +1,18 @@
+name: Build branch
+on: [push]
```

Comment:
Shouldn't it run builds only for pull-request and "deployed branches"?

Reply:
Nope all branches including master.

I think it's better this way because we want to build master on the 3 Java versions. Moreover it runs concurrently so deployment doesn't take more time.

Reply:
No I mean: avoiding to build any time a new branch is pushed.
Example:
- I push a new branch `feature/job`: I don't want to trigger a build for this branch
- When I create a pull-request from this branch, then I want to trigger the build process

Reply:
I like to start a build for my branches even if I don't create a PR. But it's a matter of taste.

Reply:
That's more like a job for your local build before creating a pull-request. And it's still useless resource consumption.

Anyway